docs/notebooks/gis.ipynb

```diff
@@ -0,0 +1,426 @@
+{
```

Comment:
<p>for the documentation<code><strong>,</strong></code> as it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>